spec/presenters/c2_status_presenter/c2_status_presenter_factory_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe C2StatusPresenterFactory do
```

Comment:
having unit tests for this is great! I don't think it is 100% necessary since the presenters are all tested via feature specs, but never bad to have more coverage ❤️